test/integration_test.py

```diff
@@ -461,7 +461,8 @@ def _wait_for_indexer():
         for catalog in catalogs:
             self._test_manifest(catalog.name)
             self._test_dos_and_drs(catalog.name)
-            self._test_repository_files(catalog.name)
+            if not config.is_anvil_enabled(catalog.name):
```

Comment:
```suggestion
            # FIXME: File downloads are broken on anvil deployments
            #        https://github.com/DataBiosphere/azul/issues/4507
            if not config.is_anvil_enabled(catalog.name):
```

Reply:
Good catch!

Reply:
Dropped this change since it was already applied by #4479